src/PrestaShopBundle/Form/Admin/Type/TypeaheadProductCollectionType.php

```diff
@@ -50,6 +52,7 @@ public function __construct($productAdapter, $categoryAdapter)
     {
         $this->productAdapter = $productAdapter;
         $this->categoryAdapter = $categoryAdapter;
+        $this->configuration = new Configuration();
```

Comment:
Use the parameter sent by form.type.typeahead.product maybe?

Reply:
Fixed that